app/models/gi_bill_feedback.rb

```diff
@@ -100,6 +100,10 @@ def save
 
   private
 
+  def anonymous?
+    parsed_form['onBehalfOf'] == 'Anonymous'
+  end
+
   def transform_keys_into_array(hash)
     array = []
     return array if hash.blank?
```

Comment:
Unrelated to the PR, but I noticed that these first two lines of the method are unnecessary.